### ISSUE TITLE
Getting Started - Promises

### ISSUE THREAD

**Issue by FletcherJMoore** [FletcherJMoore/DeckVault#3]
# User Story
Now that we have tested our api calls we will need to make some Promises that will come in handy later to get the data from our database. 

# Acceptance Criteria
WHEN you open vscode or github you will see a folder labeled api
THEN if you open that folder you will see a file labeled CardData
AND if you open that file it will show a series of Promises for every action the user will be able to do on the page.

# Dependencies
For this ticket you will need to create some promises for us to user later on. For this ticket to be considered complete you must: 
- Create a API folder in vscode
- create a file in the api folder under the name CardData
- Create the promises for you page. The names of the promises will be:  getCards, getMyCards, createCard, updateCard, deleteCard.

Here is an example of what we are looking for in those promises (Note: Not all of the promises should look exactly like this. Take note of what promises are passing a firebasekey or uid)

![Screenshot 2024-05-20 183052](https://github.com/FletcherJMoore/DeckVault/assets/154636617/54beb86d-cda8-4e2e-a974-0db870f99bd4)

# Dev Notes
For this ticket you will need to really think through what your application will need to function. Remember we are only coding promises for MVP so do not start coding stretch goals until you have completed all the MVP tickets with no errors. If you need help don't forget you can submit a discussion ticket.

**Comment by FletcherJMoore** [comment]
All Promises made!
